.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test
```

Comment:
A bit nit picky, but do we want to more specifically name this so the display in the Actions tab is more obvious what's running here?

Reply:
This affects the badge and other direct URLs so I don't want it to be too specific because constant changing would be annoying (say if/when we add more automated tests), but certainly we can make it more specific. Any thoughts? Automated Tests?